File: assignment 10 - C# practice problems/32.cs
//Write a simple C# program to join all 
//elements of the following array into a string.

using System;
namespace MyNameSpace
{
    class Program32
    {
        public static void JoinArrayElementsIntoString()
        {

            var numArray = new[] { 1, 2, 3, 4, 5 };
            Console.WriteLine("array is " + "[{0}]", string.Join(",", numArray));
            Console.WriteLine("joined string is " + "{0}", string.Join("", numArray));


        }


    }
}

File: assignment 10 - C# practice problems/js/32.js
//Write a simple JavaScript program to join all elements of the following array into a string.
function joinArrayElementsIntoString(arrayParam) {
    return arrayParam.join("");
}

function joinArrayElementsIntoStringDriverFunction() {
    let numArray = [1, 2, 3, 4, 5];
    console.log("array is: " + numArray);
    console.log("joined array string: " + joinArrayElementsIntoString(numArray));
}

joinArrayElementsIntoStringDriverFunction();